src/libs/runtime-utils.js

```diff
@@ -10,17 +10,21 @@ import * as Style from 'src/libs/style'
 import * as Utils from 'src/libs/utils'
 
 
-export const DEFAULT_DATAPROC_DISK_SIZE = 60 // For both main and worker machine disks. Dataproc clusters don't have persistent disks.
-export const DEFAULT_GCE_BOOT_DISK_SIZE = 70 // GCE boot disk size is not customizable by users. We use this for cost estimate calculations only.
-export const DEFAULT_GCE_PERSISTENT_DISK_SIZE = 50
+export const defaultDataprocDiskSize = 60 // For both main and worker machine disks. Dataproc clusters don't have persistent disks.
+export const defaultGceBootDiskSize = 70 // GCE boot disk size is not customizable by users. We use this for cost estimate calculations only.
+export const defaultGcePersistentDiskSize = 50
```

Comment:
I'm personally not used to using camelCase for constants but since that appears to be the consensus for terra-ui, I wanted the code to be consistent.